word_guess_lv.rb

```diff
@@ -0,0 +1,312 @@
+
+# We color the program in certain messages and use features like blink and bold in them also.
+require 'colorize'
+
+
+puts "WELCOME TO DISNEY WORD GUESS"
+puts "Words Theme: Walt Disney".colorize(:blue)
+
+# one class contain different methods for the game
+class WordGuess
+
+  def initialize
+    @words_low = %w[minnie mickey daisy donald pluto goofy anna elsa alice]
+    @words_medium = %w[tigger cinderella bambi pocahontas eeyor rabbit mulan belle dumbo]
+    @words_high = %w[esmeralda quasimodo anastasia mowgli anastasia drizella kocoum maleficent]
```

Comment:
These arrays are only used in one place, in `selection_level`. That means that making them instance variables might not be appropriate, you could make them local variables in `selection_level` instead.